Makefile

```diff
@@ -2,6 +2,7 @@
 
 RUN_RUBY_CMD=docker-compose run --rm -v $$PWD:/src -w /src ruby
 CFHL_DOCKER_TAG ?= $(shell cat lib/cfhighlander.version.rb  | grep VERSION | cut -d '=' -f 2 | sed 's/\.freeze//' | sed 's/"//g')
+CFNDSL_SPEC_VERSION=34.0.0
```

Comment:
perhaps update to `CFNDSL_SPEC_VERSION ?=` so it can be passed as env variable.

Reply:
good idea.. updated PR